src/trace/main.go

```diff
@@ -53,13 +55,16 @@ type Common struct {
 	Body          string    `json:"body,omitempty"`
 	Headers       []*Header `json:"headers"`
 	Version       string    `json:"version,omitempty"`
+	Time          int64     `json:"time,omitempty"`
 }
 
 type Header struct {
 	Key   string `json:"key,omitempty"`
 	Value string `json:"value,omitempty"`
 }
 
+var nativeEndian binary.ByteOrder
```

Comment:
why it is a global var and not something on the state?

Reply:
Since I need it in global context. I can put it on the vmContext and reassign it on every new http context, but I wanted to avoid it